Source/player.cpp

```diff
@@ -265,6 +265,16 @@ void PlayerStruct::RemoveInvItem(int iv, bool calcScrolls)
 		CalcScrolls();
 }
 
+bool PlayerStruct::FindAndRemoveInvItem(int item)

```

Comment:
I think `int item` is very misleading. I'd expect an element named `item` to be of type `ItemStruct` instead.

Can we rename it to be more specific? `itemId` maybe?

Reply:
no, I literally copied it from the function above lol

Reply:
Screw the function above 🤣 

So much of the code needs work that we can't just blindly copy something and consider that as good IMHO. New code should follow stricter standards.